templates/zerver/api/outgoing-webhooks.md

```diff
@@ -37,25 +37,38 @@ There are currently two ways to trigger an outgoing webhook:
 
 The Zulip-format webhook messages post the following data, encoded as JSON:
 
-```
-data:       content of message in a ready-to-use format (with the mention stripped)
-token:      A string of alphanumeric characters you can use to
-            authenticate the webhook request (each bot user uses a fixed token).
-message:    the message which triggered outgoing webhook
-├── id
-├── sender_email
-├── sender_full_name
-├── sender_short_name
-├── sender_realm_str
-├── sender_id
-├── type
-├── display_recipient
-├── recipient_id
-├── subject
-├── timestamp
-├── avatar_url
-├── client
-```
+<table class="table">
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>bot_email</code></td>
+            <td>Email of the bot user</td>
+        </tr>
+        <tr>
+            <td><code>data</code></td>
+            <td>Content of the message in a ready-to-use format (with the
+            mention stripped)</td>
+        </tr>
+        <tr>
+            <td><code>message</code></td>
+            <td>The message which triggered the outgoing webhook</td>
```

Comment:
I feel like this should probably have more context, e.g. "A dict containing details on the message".  And there's some fields we'll probably want to explain, like `recipient_id` (which you can use as an ID for the stream that will persist across the stream being renamed).